api/developerTestAccounts.ts

```diff
@@ -88,6 +89,36 @@ export function fetchDeveloperTestAccountData(
   return axios<DeveloperTestAccount>(reqWithToken);
 }
 
+export function installOauthAppIntoDeveloperTestAccount(
+  accountId: number,
+  testAccountId: number,
+  projectName: string,
+  appUId: string
+): HubSpotPromise<InstallOauthAppIntoDeveloperTestAccountResponse> {
+  return http.post<InstallOauthAppIntoDeveloperTestAccountResponse>(accountId, {
+    url: `${TEST_ACCOUNTS_API_PATH_V3}/install-apps`,
+    data: {
+      testPortalId: testAccountId,
+      developerQualifiedSymbol: { developerSymbol: appUId, projectName },
+    },
+    timeout: SANDBOX_TIMEOUT,
+  });
+}
+
+export function fetchDeveloperTestAccountOauthAppInstallStatus(
+  accountId: number,
+  projectName: string,
+  appUId: string
+): HubSpotPromise<{ status: 'IN_PROGRESS' | 'SUCCESS' }> {
```

Comment:
Nit: We should have constants for these

Reply:
Also, can this fail?

Reply:
It actually can't ever return a fail status. The endpoint only returns in_progress or success statuses. It's something to do with the service being a wrapper around some other functionality (I don't fully understand)

But good call on the types. I'll add that 👍